lib/activator.js

```diff
@@ -177,11 +178,12 @@ module.exports = {
 		templates = config.templates || DEFAULTS.templates;
 		resetExpire = config.resetExpire || DEFAULTS.resetExpire;
 		proto = config.protocol || DEFAULTS.proto;
-		mailer = smtp(transport,templates, config.attachments);
+		mailer = smtp(transport,templates, config.styliner);
```

Comment:
I really have no clue how it ended up in that line. It doesn't make sense anyways, since attachments only are passed at the `activate` or `resetPassword` call. And you are right, it is not used there at all.